data/en/imagefonts.json

```diff
@@ -4,11 +4,11 @@
 	"syntax": "imageFonts()",
 	"returns": "array",
 	"related": [],
-	"description": "",
+	"description": "return all available",
```

Comment:
All available what?

Reply:
All available image fonts.  Right?  I pulled that from the Lucee documentation.

Reply:
yup. If you change it to "Returns all available image fonts", it's fine ;-)